controllers/configmap_controller.go

```diff
@@ -258,9 +274,14 @@ func hasAssociatedInstance(nodeAddresses []core.NodeAddress, instances []*instan
 
 // mapToConfigMap fulfills the MapFn type, while always returning a request to the windows-instance ConfigMap
```

Comment:
Please update the comment

Reply:
I have a question if I am understanding this change correctly. We feed `mapToConfigMap` into the `EnqueueRequestsFromMapFunc` function. Does returning a slice with 2 `Requests` in it mean that, for every event on either ConfigMap, 2 Requests will be enqueued?

Reply:
I would think so. You might need to make this mapping function more smart. BTW, what do you see in your testing?

Reply:
I don't see 2 Requests being processed in my testing with this change. 
Triggering an event on the services configmap (deleting it) results in a single request processed :
```
1.6480679313255267e+09 INFO controllers.configmap Created {"ConfigMap": "openshift-windows-machine-config-operator/windows-services-5.0.0-9419065-dirty"}
1.648067931325568e+09 DEBUG controllers.configmap Reacting to SERVICES event...
```

Likewise, triggering an event on the instances configmap (adding a bogus entry) results in a single request processed: 
```
1.648068484640567e+09 DEBUG controllers.configmap Reacting to INSTANCES event...
1.6480684846406677e+09 ERROR controller.configmap Reconciler error {"reconciler group": "", "reconciler kind": "ConfigMap", "name": "windows-instances", "namespace": "openshift-windows-machine-config-operator", "error": "unable to parse instances from ConfigMap: unable to get username for example.property.1: data has an incorrect format",...
```

Suffice to say I am confused what the purpose of the `[handler.EnqueueRequestsFromMapFunc(r.mapToConfigMap)](https://github.com/openshift/windows-machine-config-operator/pull/1002/commits/24988bcd31186aa43836d186827a99d492d4f922#diff-f3ec2e06fa75f3747dab6fac751809ce87a96e02263980287e9d9477830049aaR306)` call really is.

Reply:
Roping @sebsoto into this convo -- I am not convinced [dropping the change](https://github.com/openshift/windows-machine-config-operator/pull/1002#discussion_r833554476) to `mapToConfigMap` is sufficient

Reply:
The purpose is triggering a windows instance configmap reconciliation upon BYOH Node events

Reply:
[Here](https://github.com/openshift/windows-machine-config-operator/pull/1002/files#r833713522) you're not seeing reconciliations for both because you are not hitting this function. this function is only called when BYOH Windows node events occur